Makefile

```diff
@@ -219,7 +231,8 @@ check-licenses: ## Checks license status.
 check-prereqs: ## Checks prerequisite software status.
 	./scripts/prereq-check.sh
 
-check-style: govet gofmt check-licenses ## Runs govet and gofmt against all packages.
+# TODO: move govet and gofmt checks inside golangci-lint once it starts running without --new-from-rev.
+check-style: govet gofmt golangci-lint check-licenses ## Runs govet, gofmt and golangci-lint against all packages.
```

Comment:
I talked with our admins and they enabled the SAS version of golang-ci. You can see the current report at https://golangci.com/r/github.com/mattermost/mattermost-server. Hence, we don't see to include this into your pipeline. Would you just remove the check from `check-style` for now?